### ISSUE TITLE
Problem  with anvi-setup-scg-databases

### ISSUE THREAD

**Issue by razz1618** [merenlab/anvio#1488]
I am trying to run anvi-setup-scg-databases
 for the first time on ubuntu 18 but it fails with following code

anvi-setup-scg-databases

```
WARNING
===============================================
Please remember that the data anvi'o uses for SCG taxonomy is a courtesy of The
Genome Taxonomy Database (GTDB), an initiative to establish a standardised
microbial taxonomy based on genome phylogeny, primarly funded by tax payers in
Australia. Please don't forget to cite the original work, doi:10.1038/nbt.4229
by Parks et al to explicitly mention the source of databases anvi'o relies upon
to estimate genome level taxonomy. If you are not sure how it should look like
in your methods sections, anvi'o developers will be happy to help you if you
can't find any published example to get inspiration.

                                                                                [12 Aug 20 22:04:44 Creating search databases] Removing any database that still                                                                                 [12 Aug 20 22:04:44 Creating search databases] Copying FASTA files to /tmp/tmp_233pylm ...                             Traceback (most recent call last):
  File "/usr/local/bin/anvi-setup-scg-databases", line 43, in <module>
    s.setup()
  File "/usr/local/lib/python3.6/dist-packages/anvio/scgtaxonomyops.py", line 1879, in setup
    self.create_search_databases()
  File "/usr/local/lib/python3.6/dist-packages/anvio/scgtaxonomyops.py", line 1999, in create_search_databases
    new_paths = dict([(os.path.basename(fasta_path), shutil.copy((fasta_path + '.gz'), os.path.join(temp_dir, os.path.basename(fasta_path) + '.gz'))) for fasta_path in [s['fasta'] for s in self.ctx.SCGs.values()]])
  File "/usr/local/lib/python3.6/dist-packages/anvio/scgtaxonomyops.py", line 1999, in <listcomp>
    new_paths = dict([(os.path.basename(fasta_path), shutil.copy((fasta_path + '.gz'), os.path.join(temp_dir, os.path.basename(fasta_path) + '.gz'))) for fasta_path in [s['fasta'] for s in self.ctx.SCGs.values()]])
  File "/usr/lib/python3.6/shutil.py", line 245, in copy
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/usr/lib/python3.6/shutil.py", line 120, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: '/usr/local/lib/python3.6/dist-packages/anvio/data/misc/SCG_TAXONOMY/GTDB/SCG_SEARCH_DATABASES/Ribosomal_S2.gz'
``



when i try to reset, i get this error message
 anvi-setup-scg-databases --reset

```
WARNING
===============================================
The existing directory for SCG taxonomy data dir has been removed. Just so you
know.

WARNING
===============================================
Please remember that the data anvi'o uses for SCG taxonomy is a courtesy of The
Genome Taxonomy Database (GTDB), an initiative to establish a standardised
microbial taxonomy based on genome phylogeny, primarly funded by tax payers in
Australia. Please don't forget to cite the original work, doi:10.1038/nbt.4229
by Parks et al to explicitly mention the source of databases anvi'o relies upon
to estimate genome level taxonomy. If you are not sure how it should look like
in your methods sections, anvi'o developers will be happy to help you if you
can't find any published example to get inspiration.

Local directory to setup .....................: /usr/local/lib/python3.6/dist-packages/anvio/data/misc/SCG_TAXONOMY/GTDB
Reset the directory first ....................: True

Remote database ..............................: GTDB
Remote URL to download files .................: https://data.ace.uq.edu.au/public/gtdb/data/releases/latest/
Remote files of interest .....................: VERSION, ar122_msa_individual_genes.tar.gz, ar122_taxonomy.tsv, bac120_msa_individual_genes.tar.gz, bac120_taxonomy.tsv
                                                                                [12 Aug 20 21:28:12 GTDB setup] Reading the VERSION file...                                                                                                     GTDB release found ...........................: v95 (Released July 17, 2020)
                                                                                [12 Aug 20 21:28:15 Downloading "VERSION"] ...                                                                                                                  [12 Aug 20 21:28:15 Downloading "VERSION"] 100.0%                                                                                                               Downloaded succesfully .......................: /usr/local/lib/python3.6/dist-packages/anvio/data/misc/SCG_TAXONOMY/GTDB/VERSION

Config Error: Something went wrong with your download attempt. Here is the problem: 'HTTP
              Error 404: Not Found'                                                      

```


I guess there might be some problem while downloading the required files as when i open "/usr/local/lib/python3.6/dist-packages/anvio/data/misc/SCG_TAXONOMY/GTDB" directory i can see only one file named Version. Plus, the file names " at https://data.ace.uq.edu.au/public/gtdb/data/releases/latest/" does not match with ANVIO's Remote files of interest .....................: VERSION, ar122_msa_individual_genes.tar.gz, ar122_taxonomy.tsv, bac120_msa_individual_genes.tar.gz, bac120_taxonomy.tsv

**Comment by meren** [comment]
Hi @razz1618, yes, the file names have changed in the latest release of GTDB, and anvi'o was not prepared for that. We fixed it in #1475, but it is only available to those who follow the `master` branch (a way to do that is explained in our installation manual).

If you reinstall anvi'o through conda, you will have the necessary files to run `anvi-setup-scg-databases` without any issues. But `anvi-setup-scg-databases --reset` will not work.

In anvi'o v7 which we plan to release within the next few months will not have this issue.

Apologies for the inconvenience.

**Comment by razz1618** [comment]
Noted. Thanks for the good work.
